### DISCUSSION TITLE
[218620] PAYDAY 2   [本体]

### DISCUSSION THREAD

**Discussion by an040307** [pjy612/SteamManifestCache#539] [Share]
`appid:218620`
 keys:
 ```
 218631;1c394d148b286577ab96453164cb03ded6619ccfad3da0f01aac23d71302d1e2
 218621;2d0de203756909d9b2b65506594648ce773a0428741fcae5208616bd9c79d3ea
 ```
 
[[218620]payday2.zip](https://github.com/pjy612/SteamManifestCache/files/14536880/218620.payday2.zip)

 code:CE58985DEB0E9011
 email:3422752856@qq.com

**Comment by pjy612** [comment]
* 烦请调整下格式 包括注意中英文标点 #26
* 貌似额外的中文配置 218631 https://steamdb.info/app/218620/depots/
* 仅分享 没有别的东西，有必要 可以帮忙删帖

**Reply by an040307** [reply]
你看看对了吗

**Reply by pjy612** [reply]
格式✔
